bitacross-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -208,7 +210,8 @@ impl TrustedCallVerification for TrustedCallSigned {
 		payload.append(&mut mrenclave.encode());
 		payload.append(&mut shard.encode());
 
-		self.signature.verify(payload.as_slice(), self.call.sender_identity())
+		self.signature.verify(&blake2_256(&payload), self.call.sender_identity())
```

Comment:
I would say we keep only one way of signing message, that is hashing it before we sign it. I always assumed the sign and verify methods did the hashing internally 😅

Reply:
Yea this is changed in tee-worker already, just not in bc-worker.